app/components/product_component/product_component.html.haml

```diff
@@ -0,0 +1,11 @@
+%tr
+  - @columns.each do |column|
+    - if column[:id] == "name"
+      %td.products_column.title
+        - if @image
+          .image
+          = image_tag @image.url(:mini)
+        = @name
```

Comment:
This is looks good. 

Just wondering if it'll be better to add a case for `name`
https://github.com/openfoodfoundation/openfoodnetwork/blob/500c4e8a2f5fd268619e60fb47581cb6185fb2f1/app/components/product_component.rb#L17-L28
then you'll add one guard clause for the image, this will maybe require some CSS changes, something like:
```haml
%td.products_column{class: column[:id]}
        - if column[:id] == "name" && @image
          .image
          = image_tag @image.url(:mini)
        = column[:value]
```